labs/lab-1/FillOneRegularPothole.k

```diff
@@ -0,0 +1,26 @@
+function fillpotholes () { 
+	repeat (5) { 
```

Comment:
You might consider using a `while` loop rather here instead and that could be based on (and also replace) the `if` condition you have within the loop.